lib/streamlit/runtime/runtime.py

```diff
@@ -554,12 +557,15 @@ async def _loop_coroutine(self) -> None:
                     return_when=asyncio.FIRST_COMPLETED,
                 )
 
-            # Shut down all AppSessions.
-            # NOTE: We want to fully shut down sessions when the runtime stops for now,
-            # but this may change in the future if/when our notion of a session is no
-            # longer so tightly coupled to a browser tab.
-            for session_info in self._session_mgr.list_sessions():
-                self._session_mgr.close_session(session_info.session.id)
+            # Shut down all AppSessions. The trailing underscore in session_info_ is
+            # needed because session_info (with no underscore) above is bound to an
+            # ActiveSessionInfo, which is a type mismatch with the SessionInfo we get
+            # back from list_sessions() here.
```

Comment:
I don't feel strongly ,but I wonder if it's worth renaming the above `session_info` to `active_session` to avoid this minor awkwardness

Reply:
Ah, yeah the rename is far preferable to the weird trailing underscore and explanatory comment. Let me change that